.github/workflows/tests.yml

```diff
@@ -200,7 +200,7 @@ jobs:
       matrix:
         # test all lower versions (w/ defaults) and upper version (w/ defaults and conda-forge)
         runs-on: [ubuntu-latest, ubuntu-24.04-arm]
```

Comment:
Hm should we limit ARM jobs to only run with Python 3.13? Running it for all Python versions results in a lot of jobs.